pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -434,6 +434,17 @@ func (k *Bootstrapper) UpdateCluster(cfg config.MachineConfig) error {
 	if err := bsutil.AddAddons(&files, assets.GenerateTemplateData(cfg.KubernetesConfig)); err != nil {
 		return errors.Wrap(err, "adding addons")
 	}
+
+	// Combine mkdir request into a single call to reduce load
+	dirs := []string{}
+	for _, f := range files {
+		dirs = append(dirs, f.GetTargetDir())
+	}
+	args := append([]string{"mkdir", "-p"}, dirs...)
+	if _, err := k.c.RunCmd(exec.Command("sudo", args...)); err != nil {
```

Comment:
how about we run all of them in one command ? (each invokation of a cmmand over ssh or docker exec take a bit time)
```
 sudo mkdir /tmp/1  /tmp/2
```
instead of
``` 
sudo mkdir /tmp/1  
sudo mkdir /tmp/2

```

Reply:
oopps never mind this is already doing that